tools/kafka/loopback/README.md

```diff
@@ -0,0 +1,50 @@
+# 100K Loopback
+
+This service is a helper for 100k-epic, transfers incoming payload from a **device-data** topic to **device-manager** topic in the dojot style.
+
+# Configurations
+## Environment Variables
+
+Key                      | Purpose                                                             | Default Value   			| Valid Values      |
+------------------------ | ------------------------------------------------------------------- | -------------------------- | ----------------- |
+DOJOT_USERNAME           | username to login on auth and retrieve token						   | admin           			| string   		    |
+DOJOT_PASSWORD           | password to login on auth and retrieve token						   | admin           			| string   		    |
+AUTH_ADDRESS             | Address of the auth service                                         | http://auth:5000           | hostname/IP:port  |
+DATA_BROKER_ADDRESS      | Address of the data broker service                                  | http://data-broker:80    	| hostname/IP:port  |
+KAFKA_BROKER_LIST        | Addresses of the kafka brokers separated by a comma                 | kafka-server:9092			| hostname/IP:port  |
+LOOPBACK_CONSUMER_GROUP  | Kafka consumer group                                                | loopback-group             | string            |
+DEVICE_DATA_TOPIC    	 | Topic to consume from                                               | device-data        		| string            |
+DEVICE_MANAGER_TOPIC     | Topic to produce the modified messages                              | dojot.device-manager.device| string            |
+
+# Example
+As a specific component for dojot devices there is an example
```

Comment:
Bellow there is an example of the message received from the **device-data** topic and the modified message sent to **device-manager** topic.